src/actions/guides/database/database_setup.cr

```diff
@@ -0,0 +1,39 @@
+class Guides::Database::DatabaseSetup < GuideAction
+  guide_route "/database/database-setup"
+
+  def self.title
+    "Database Setup"
+  end
+
+  def markdown
+    <<-MD
+    ## Configure
+
+    Maybe a mention of which version of postgres needs to be installed or min version?
+
+    Mention `config/database.cr` and how to configure connection and Avram configure options
+
+    > Note about CLI tools for postgres app on macOS.
+
+    ### Test setup
+
+    Notes about using .env so test DB doesn't conflict with dev DB
```

Comment:
This is done with `config/database.cr` and not `.env` Might be worth mentioning that.

Also might want to add a section called `Setting local credentials` in which case you'd want to use `config/local/database.cr` where you can override the settings. Or use `.env`